python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -132,43 +132,79 @@ def __call__(self, df):
 
 
 class CalcE(MultiColumnAction):
```

Comment:
Should this class `CalcE` and the others be listed in the `__all__` at the top of the file?

Reply:
Good catch, Andrés!